src/main/java/org/opensearch/flowframework/workflow/CreateIndexStep.java

```diff
@@ -116,140 +101,140 @@ public void onFailure(Exception e) {
-                                            );
-                                        }
-                                    }, exception -> {
-                                        logger.error("Failed to update index " + indexName, exception);
-                                        internalListener.onFailure(exception);
-                                    })
-                                );
-                        } else {
-                            // no need to update index if it does not exist or the version is already up-to-date.
-                            indexMappingUpdated.get(indexName).set(true);
-                            internalListener.onResponse(true);
-                        }
-                    }, e -> {
-                        logger.error("Failed to update index mapping", e);
-                        internalListener.onFailure(e);
-                    }));
-                } else {
-                    // No need to update index if it's already updated.
-                    internalListener.onResponse(true);
-                }
-            }
-        } catch (Exception e) {
-            logger.error("Failed to init index " + indexName, e);
-            listener.onFailure(e);
-        }
-    }
-
-    /**
-     * Get index mapping json content.
-     *
-     * @param mapping type of the index to fetch the specific mapping file
-     * @return index mapping
-     * @throws IOException IOException if mapping file can't be read correctly
-     */
-    public static String getIndexMappings(String mapping) throws IOException {
-        URL url = CreateIndexStep.class.getClassLoader().getResource(mapping);
-        return Resources.toString(url, Charsets.UTF_8);
-    }
-
-    /**
-     * Check if we should update index based on schema version.
-     * @param indexName index name
-     * @param newVersion new index mapping version
-     * @param listener action listener, if update index is needed, will pass true to its onResponse method
-     */
-    private void shouldUpdateIndex(String indexName, Integer newVersion, ActionListener<Boolean> listener) {
-        IndexMetadata indexMetaData = clusterService.state().getMetadata().indices().get(indexName);
-        if (indexMetaData == null) {
-            listener.onResponse(Boolean.FALSE);
-            return;
-        }
-        Integer oldVersion = NO_SCHEMA_VERSION;
-        Map<String, Object> indexMapping = indexMetaData.mapping().getSourceAsMap();
-        Object meta = indexMapping.get(META);
-        if (meta != null && meta instanceof Map) {
-            @SuppressWarnings("unchecked")
-            Map<String, Object> metaMapping = (Map<String, Object>) meta;
-            Object schemaVersion = metaMapping.get(SCHEMA_VERSION_FIELD);
-            if (schemaVersion instanceof Integer) {
-                oldVersion = (Integer) schemaVersion;
-            }
-        }
-        listener.onResponse(newVersion > oldVersion);
-    }
+    // /**
```

Comment:
Please try to avoid block commenting-out with `//`.  It ends up creating a diff with the entire code, removing indentation, and making it difficult to see if/when anything is changed internally when uncommenting later.

Using the special format `/*-` will [preserve all the original format inside the comment](https://stackoverflow.com/questions/1024886/making-eclipses-java-code-formatter-ignore-block-comments) (in the Eclipse formatter, which is what spotless will use).